tests/phpunit/tests/test-sync.php

```diff
@@ -263,8 +271,10 @@ function test_save_post_with_sync() {
 		$key = add_post_meta( $from, 'key', 'value' );
 		$metas[ $key ] = array( 'key' => 'key', 'value' => 'value' );
 
+		$this->pll_admin->default_term = new PLL_Admin_Default_Term( $this->pll_admin );
```

Comment:
This is already done earlier in the test. You are setting the filters and action a second time here.

Reply:
Oops, I hadn't noticed.